template/web/src/components/layout/banner/banner.tsx

```diff
@@ -5,14 +5,17 @@ import NextLink from 'next/link';
 type BannerProps = {
   pageName: string;
   pageUrl: string;
+  wip?: boolean;
 };
 
-export default function Banner({ pageName, pageUrl }: BannerProps) {
+export default function Banner({ pageName, pageUrl, wip }: BannerProps) {
   return (
     <section
       className={clsx(
         'flex flex-col items-center justify-between gap-6 p-6 md:flex-row md:gap-0',
-        'rounded-lg border border-zinc-400 border-opacity-20 bg-neutral-900',
+        `rounded-lg border border-zinc-400 border-opacity-10  ${
```

Comment:
nit: Don't need to use a template string literal here. With clsx we can just use a standard ternary with string values.

Reply:
Done!